From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: bsp <bsp@radxa.com>
Date: Wed, 26 Apr 2023 14:51:21 +0800
Subject: [PATCH] Update file loading address


0x02800000 currently will be overwritten by kernel_addr_r
on Radxa system (end at 0x02A3EA00), so move it out of this
memory region.

Also add fdtoverlay_addr_r to enable loading overlays.
---
 include/configs/rk3308_common.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/rk3308_common.h b/include/configs/rk3308_common.h
index 9e5de8e1a0..79cd116580 100644
--- a/include/configs/rk3308_common.h
+++ b/include/configs/rk3308_common.h
@@ -16,9 +16,12 @@
 #define ENV_MEM_LAYOUT_SETTINGS \
 	"scriptaddr=0x00500000\0" \
 	"pxefile_addr_r=0x00600000\0" \
-	"fdt_addr_r=0x02800000\0" \
+	"fdt_addr_r=0x1d000000\0" \
+	"fdtoverlay_addr_r=0x1f000000\0" \
 	"kernel_addr_r=0x00680000\0" \
- 	"ramdisk_addr_r=0x04000000\0"
+	"ramdisk_addr_r=0x04000000\0" \
+	"kernel_comp_addr_r=0x10000000\0" \
+	"kernel_comp_size=0x2000000\0"
 
 #include <config_distro_bootcmd.h>
 #define CFG_EXTRA_ENV_SETTINGS \
-- 
2.40.0

